docs/source/connecting/tls.md

```diff
@@ -59,7 +59,7 @@ context_builder.set_verify(SslVerifyMode::PEER);
 
 let session: Session = SessionBuilder::new()
     .known_node("127.0.0.1:9142") // The the port is now 9142
-    .ssl_context(Some(context_builder.build()))
+    .tls_context(Some(context_builder.build()))
     .build()
     .await?;
 
```

Comment:
Commit: "codewide: wrap openssl usage with generic TLS wrappers "

❓ This may become clear later but a few lines later I see:
`.tls_context(Some(TlsContext::OpenSsl010(context_builder.build())))`

Why the difference?

Reply:
I've made both lines the shorter version (without mentioning `TlsContext`).